tfhe/docs/getting_started/benchmarks/gpu/gpu_integer_operations.md

```diff
@@ -6,7 +6,7 @@ This document details the GPU performance benchmarks of homomorphic operations o
 All GPU benchmarks were launched on H100 GPUs, and rely on the multithreaded PBS algorithm.
 {% endhint %}
 
-The cryptographic parameters `PARAM_GPU_MULTI_BIT_MESSAGE_2_CARRY_2_GROUP_3_KS_PBS` were used.
+The cryptographic parameters `PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS` were used.
```

Comment:
You're sure about that ? I don't recall using the the grouping factor 4 on this benchmarks.

Reply:
it's planned to be used @agnesLeroy do you have the benchmarks for those ?